.github/workflows/main.yml

```diff
@@ -16,18 +17,23 @@ jobs:
         - ubuntu-latest
         - macos-latest
         - windows-latest
+        include:
+        - python: "os"
+          platform: ubuntu-latest
+          toxargs: "--sitepackages"
     runs-on: ${{ matrix.platform }}
     steps:
       - uses: actions/checkout@v3
       - name: Setup Python
+        if: ${{ matrix.python != 'os' }}
         uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python }}
       - name: Install tox
         run: |
           python -m pip install tox
       - name: Run tests
-        run: tox
+        run: tox ${{ matrix.toxargs }}
```

Comment:
I'd prefer if some setting in the environment would be sufficient, rather than having to plumb this setting through to the invocation. I've filed https://github.com/tox-dev/tox/issues/2473 to request that feature.

Reply:
Sounds good